src/widget/wcuemenupopup.h

```diff
@@ -59,6 +54,8 @@ class WCueMenuPopup : public QWidget {
     void slotChangeCueColor(mixxx::RgbColor::optional_t color);
 
   private:
+    void closeEvent(QCloseEvent* event) override;
```

Comment:
This method is protected in the base class and you should use the same visibility here. Reducing the visibility in derived classes is not recommended, because it does not work as intended.